<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryEmp</class>
 <widget class="QDialog" name="QueryEmp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>员工查询</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>员工查询</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6,2">
     <item>
      <widget class="QComboBox" name="cbbType"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leKey"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuery">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
